packages/common/src/projects/HubProjectManager.ts

```diff
@@ -161,30 +172,52 @@ export class HubProjectManager implements IHubEntityManager<IHubProject> {
     requestOptions?: IRequestOptions
   ): Promise<IHubProject> {
     if (requestOptions || this.context.requestOptions) {
-      return getProject(
+      return fetchProject(
         identifier,
         requestOptions || this.context.requestOptions
       );
     } else {
       throw new HubError(
-        "Get Project",
+        "Fetch Project",
```

Comment:
What's the pattern for get vs fetch? Just async?

Reply:
Correct. as per the naming docs we created a while back, `fetch` implies async, where as `get` is sync -> https://esri.github.io/hub-components/?path=/story/guides-naming--page (unf in-page anchors don't work in storybook)